worker/uniter/runner/context/env.go

```diff
@@ -34,6 +48,19 @@ func OSDependentEnvVars(paths Paths, getEnv GetEnvFunc) []string {
 	return nil
 }
 
+// PlatformEnvVars provides a quick way to detect any platform specific env vars
+// that should be used by the caller. This was originally written to pass
+// through Kubernetes env vars.
+func PlatformEnvVars(getEnv GetEnvFunc) []string {
+	pVars := []string{}
+	for _, e := range platformEnvKeys {
```

Comment:
Probably ok as a quick fix but we are changing the behaviour of a released juju, ie juju run now would no longer include all env vars like it did with os.Environ(). Not sure if we shouldn't retain the existing juju run behaviour. Also, by filtering just an KUBERNETES_ vars, we would be potentially suppressing any env vars juju might in the future apply to the container config in the pod spec?